uint8 palette
---
bool result
